src/microbe_stage/MicrobeHUD.cs

```diff
@@ -542,12 +566,58 @@ protected override void Dispose(bool disposing)
                 IngestedMatterBarPath.Dispose();
                 BindingModeHotkeyPath.Dispose();
                 UnbindAllHotkeyPath.Dispose();
+                StrainBarPanelPath.Dispose();
+                StrainBarPath.Dispose();
+                StrainBarFadeAnimationPlayerPath.Dispose();
             }
         }
 
         base.Dispose(disposing);
     }
 
+    private void UpdateStrain(Entity player)
+    {
+        if (!stage!.GameWorld.WorldSettings.ExperimentalFeatures)
+        {
+            strainBar.Visible = false;
+        }
+
+        var strainFraction = player.Get<StrainAffected>().CalculateStrainFraction();
```

Comment:
This should check if the component exists and if not print an error *once* and then do nothing. This will ensure compatibility with older saves and rare bugs if the component is somehow missing. This isn't a critical feature to display the info so it missing shouldn't crash the game.